### PR TITLE
Add schema diagram and create table SQL query

### PR DESCRIPTION
**Pull request by togimoto**

### CODE REVIEW

File: db/create_tables.sql [hunk 1]
```diff
@@ -0,0 +1,96 @@
+CREATE TABLE "USERS" (
+  "ntuEmail" varchar(63) UNIQUE NOT NULL,
+  "username" varchar(80) PRIMARY KEY,
```
Comment (yenting-biao) [inline db/create_tables.sql:3]: USERS 的 PK 是 ntumail
username 是要給 user 亂取的暱稱
然後可能需要一個 avatar url

**Review by yenting-biao — CHANGES_REQUESTED** [review]
please do it like 網服作業

**Comment by nukkk97** [comment]
waits for later change

**Review by nukkk97 — APPROVED** [review]
add schema later

**Review by yenting-biao — APPROVED** [review]
Remember to add schema.ts and all the related things with db (including docker if needed) later

**Pull request merged**